src/ipyforcegraph/behaviors/_base.py

```diff
@@ -0,0 +1,31 @@
+""" Base classes for behaviors.
+"""
```

Comment:
Unnecessary new line.  Only mentioning it in case there are other changes that warrant a new commit.

Reply:
2329dac